src/Core/Models/Business/OrganizationLicense.cs

```diff
@@ -230,35 +230,52 @@ public byte[] ComputeHash()
 
     public bool CanUse(IGlobalSettings globalSettings, ILicensingService licensingService, out string exception)
     {
-        if (!Enabled || Issued > DateTime.UtcNow || Expires < DateTime.UtcNow)
+        var errorMessages = new StringBuilder();
+
+        if (!Enabled)
         {
-            exception = "Invalid license. Your organization is disabled or the license has expired.";
-            return false;
+            errorMessages.AppendLine("The license is not enabled.");
```

Comment:
⛏️ Sorry, just one more nit that I missed before. `!Enabled` signifies that the cloud-hosted organization is disabled, not that the license is disabled.

This could be "Your cloud-hosted organization is currently disabled."